### PR TITLE
#165020074 update travis CI to run tests

### PR DESCRIPTION
**Pull request by Karlmusingo**
#### What does this PR do?
Configure travisCI to run the tests
#### Description of Task to be completed?
~ write the configuration of travisCI in .travis.yml
#### How should this be manually tested?
Go to [travisCI ah-kgl-wakanda-frontend](https://travis-ci.com/andela/ah-kgl-wakanda-frontend)
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#165020074](https://www.pivotaltracker.com/story/show/165020074)
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A

### CODE REVIEW

**Review by kipropbrian — CHANGES_REQUESTED** [review]
This PR should not have @gracelungu's commits, could you look into what the problem is?

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+language: node_js
+node_js:
+  - "11.10.1"
+cache:
+  directories:
+    - node_modules
+before_install:
+  - yarn install
+install:
+  - npm install
```
Comment (gracelungu) [inline .travis.yml]: @Karlmusingo I would like to understand why you are using Yarn to install packages and NPM one after another! Aren't they suppose to do the same thing?

**Comment by Karlmusingo** [comment]
> This PR should not have @gracelungu's commits, could you look into what the problem is?

File: .travis.yml (hunk 1 above)
Comment (Karlmusingo) [reply .travis.yml]: We keep yarn

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+install:
+  - yarn install
+script:
+  - npm test -- -u
+after_success:
+  - npm run coveralls
+  - "stable"
+notifications:
+  email: false
+  slack: true
```
Comment (dusmel) [inline .travis.yml]: Hey @Karlmusingo from this line, does that mean we have integrated Travis with our slack channel?

**Review by dusmel — CHANGES_REQUESTED** [review]

**Comment by kipropbrian** [comment]
The link to the PT board is not working. Could you link it [ #properly](https://pivotaltracker.com)

File: .travis.yml (hunk 2 above)
Comment (Karlmusingo) [reply .travis.yml]: Yes @dusmel, I've just done it and you can see notifications in our integration channel.

**Pull request merged**